test/controllers/GetTrustControllerSpec.scala

```diff
@@ -736,6 +736,72 @@ class GetTrustControllerSpec extends WordSpec with MockitoSugar
 
     }
 
+    ".getAssets" should {
+
+      "return 403 - Forbidden with parked content" in {
+
+        when(transformationService.getTransformedData(any(), any())(any()))
+          .thenReturn(Future.successful(TrustFoundResponse(ResponseHeader("Parked", "1"))))
+
+        val result = getTrustController.getAssets(utr)(FakeRequest(GET, s"/trusts/$utr/transformed/assets"))
+
+        whenReady(result) { _ =>
+          status(result) mustBe FORBIDDEN
+        }
+      }
+
+      "return 200 - Ok with processed content" in {
+
+        val processedResponse = models.get_trust.TrustProcessedResponse(getTransformedTrustResponse, ResponseHeader("Processed", "1"))
+
+        when(transformationService.getTransformedData(any[String], any[String])(any()))
+          .thenReturn(Future.successful(processedResponse))
+
+        val result = getTrustController.getAssets(utr)(FakeRequest(GET, s"/trusts/$utr/transformed/assets"))
+
+        whenReady(result) { _ =>
+          verify(mockedAuditService).audit(mockEq("GetTrust"), any[JsValue], any[String], any[JsValue])(any())
+          verify(transformationService).getTransformedData(mockEq(utr), mockEq("id"))(any())
+          status(result) mustBe OK
+          contentType(result) mustBe Some(JSON)
+          contentAsJson(result) mustBe getTransformedAssetsResponse
+        }
+      }
+
+      "return 200 - Ok with processed content with 5mld data" in {
+
+        val cached = Taxable5MLDFixtures.Cache.taxable5mld2134514321
+
+        val processedResponse = models.get_trust.TrustProcessedResponse(cached, ResponseHeader("Processed", "1"))
+
+        when(transformationService.getTransformedData(any[String], any[String])(any()))
+          .thenReturn(Future.successful(processedResponse))
+
+        val result = getTrustController.getAssets(utr)(FakeRequest(GET, s"/trusts/$utr/transformed/assets"))
+
+        whenReady(result) { _ =>
+          verify(mockedAuditService).audit(mockEq("GetTrust"), any[JsValue], any[String], any[JsValue])(any())
+          verify(transformationService).getTransformedData(mockEq(utr), mockEq("id"))(any())
+          status(result) mustBe OK
+          contentType(result) mustBe Some(JSON)
+          contentAsJson(result) mustBe Taxable5MLDFixtures.Trusts.Assets.taxable5mld2134514321Assets
+        }
+      }
+
+      "return 500 - Internal server error for invalid content" in {
+
+        when(transformationService.getTransformedData(any(), any())(any()))
+          .thenReturn(Future.successful(models.get_trust.TrustProcessedResponse(Json.obj(), ResponseHeader("Parked", "1"))))
+
+        val result = getTrustController.getBeneficiaries(utr)(FakeRequest(GET, s"/trusts/$utr/transformed/beneficiaries"))
```

Comment:
should be getAssets(utr) ...

Reply:
Yes, It'll have still worked because the thing it calls is mocked out